src/main/java/com/amashchenko/maven/plugin/gitflow/GitFlowReleaseFinishMojo.java

```diff
@@ -223,9 +223,23 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             // get current project version from pom
             final String currentVersion = getCurrentProjectVersion();
 
+            if ((commitDevelopmentVersionAtStart && useSnapshotInRelease) && ArtifactUtils.isSnapshot(currentVersion)) {
```

Comment:
This creates additional commit in production branch with snapshot version. Production branch should be clean, meaning each commit is deployable, meaning no snapshot version in production.

Reply:
But this commit is in the production branch.